lib/manifest/SmilManifestGenerator.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Please remove this file....  We will move to new stuff, no need to mainain old code